doc/source/ray-air/examples/gptj_batch_prediction.ipynb

```diff
@@ -0,0 +1,224 @@
+    "\n",
+    "We will use Ray Data to carry out this task and a pretrained model from Hugging Face hub. Note that you can easily adapt this example to use other similar models.\n",
+    "\n",
+    "```{note}\n",
+    "In order to run this example, make sure your Ray cluster has access to at least one GPU with 16 or more GBs of memory. The amount of memory needed will depend on the model.\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model_id = \"EleutherAI/gpt-j-6B\"\n",
+    "revision = \"float16\"  # use float16 weights to fit in 16GB GPUs\n",
+    "prompt = (\n",
+    "    \"In a shocking finding, scientists discovered a herd of unicorns living in a remote, \"\n",
+    "    \"previously unexplored valley, in the Andes Mountains. Even more surprising to the \"\n",
+    "    \"researchers was the fact that the unicorns spoke perfect English.\"\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import ray"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We define a runtime environment to ensure that the Ray workers have access to all the necessary packages."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ray.init(\n",
+    "    runtime_env={\n",
+    "        \"pip\": [\n",
+    "            \"accelerate>=0.16.0\",\n",
+    "            \"transformers>=4.26.0\",\n",
+    "            \"torch\",\n",
+    "        ]\n",
+    "    }\n",
+    ")"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For the purposes of this example, we will use a very small toy dataset composed of multiple copies of our prompt. Ray Data can handle much bigger datasets with ease."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import ray.data\n",
+    "import pandas as pd\n",
+    "\n",
+    "ds = ray.data.from_pandas(pd.DataFrame([prompt] * 10, columns=[\"prompt\"]))"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You may notice that we are not using an AIR {class}`Predictor <ray.train.predictor.Predictor>` here. Predictors are mainly intended to be used with AIR {class}`Checkpoints <ray.air.Checkpoint>`. Since we will be using a pretrained model from Hugging Face hub, it is simpler to instead use {meth}`map_batches <ray.data.Dataset.map_batches>` with a [callable class UDF](transform_datasets_callable_classes). This will allow us to save time by initializing a model just once and then feed it multiple batches of data."
```

Comment:
As a new user, I think this "Predictor" part is a bit confusing. Maybe leave it for the end.